### DISCUSSION TITLE
How to solve this??有人可以帮我吗？？

### DISCUSSION THREAD

**Discussion by caylanim** [windingwind/zotero-better-notes#574] [Q&A] [answered]
<img width="1011" alt="image" src="https://github.com/windingwind/zotero-better-notes/assets/139762684/4ee6458e-f117-4f24-8228-1b7f45e0bbe3">
为什么我的这个主文档图标这么多？完全做不了新笔记了。。。

**Comment by windingwind** [comment, marked as answer]
旧版本不再维护。请更新插件版本
